### DISCUSSION TITLE
Pimcore version 11: CMF: custom subfolders

### DISCUSSION THREAD

**Discussion by noolldave** [pimcore/pimcore#16542] [Q&A / Support] [answered]
Hi, 
I need to bring some order when registering new users and was thinking of using year/month as subfolders for them. Starting from the default /customers folder of CMF, I would like to have /customers/2024/01 /customers/2024/02 and save the new users accordingly. However, I can't figure out how to do this or how to get the parent folder in my controllers/services. I've tried a few google results, but most refer to old versions or @internal classes or classes marked as deprecated in the comments, but with no reference to an alternative method. 

Any ideas are welcome. Thanks in advance

**Comment by patriciomg** [comment, marked as answer]
Hi,
if you have the CMF installed, you should enable the **enableAutomaticObjectNamingScheme** and then defined what you need in **namingScheme**. If you need year and month, you should have those defined as attributes in your customer model. This is because the naming scheme use the attributes of your model.
Hope it helps you.
cheers!

**Reply by noolldave** [reply]
Thank you very much!  geez, I've read pimcore is used in thousand of companies and I wrongly assumed that at least 1% of all the users are willing to help others. I only see tumbleweeds here. Thank you again!
